doc/user/installing-llvm.md

```diff
@@ -1,6 +1,7 @@
 # Installing LLVM
 
-TruffleRuby needs LLVM to run and build C extensions - version 3.8, 3.9 or 4.0.
+TruffleRuby needs LLVM to run and build C extensions - on Linux you
+need version 3.8, 3.9 or 4.0; on macOS you will need version 4.0.
```

Comment:
This is going to conflict with #1226.

Reply:
Yeah. I'll remove this commit from the PR. and just rely on your #1226 for the documentation changes.